client/widmgr/widmgr.go

```diff
@@ -221,6 +241,11 @@ func (m *WIDMgr) getIdentities() error {
 
 		m.lastToken[id] = swid
 	}
+	// Store default identity tokens
+	for id, token := range defaultTokens {
+		m.lastToken[id] = token
+	}
```

Comment:
I think we want this to be before the previous block where we're looping over the signed identities, because that way a user can override the default.

Reply:
can users override the default?

Reply:
ah nevermind, I see it in the RFC now 🤦 I'll fix asap

Reply:
[54440e4](https://github.com/hashicorp/nomad/pull/18610/commits/54440e4c7d1dc8e69a56d3ca1b8c731a0924effd)